lib/uktt/geographical_area.rb

```diff
@@ -0,0 +1,41 @@
+module Uktt
+  class GeographicalArea
+    attr_accessor :config, :response
+
+    def initialize(opts = {})
+      Uktt.configure(opts)
+      @config = Uktt.config
+      @response = nil
```

Comment:
I mean this will be nil unless assigned anyway?

Reply:
So I guess we can use these changes as part of a wider refactor. Probably we want a base class or a module we include